### ISSUE TITLE
one-var and export minimize the readability

### ISSUE THREAD

**Issue by cristian-sima** [eslint/eslint#6774]
Let's assume two things: 
- I use [Redux](https://github.com/reactjs/redux)
- I need a lot of [action creators](http://redux.js.org/docs/basics/Actions.html) (thus a lot of action types)

In the same way I want to have `one-var` rule activated, because inside the functions I want to have a clear code with one declaration of `var`, `let` or `const`.

Therefore, I end up with this code for my `actionTypes.js` file:

``` js
const FETCH_INITIAL_INFORMATION = "FETCH_INITIAL_INFORMATION",
 FETCH_SUGGESTIONS = "FETCH_SUGGESTIONS",
 FETCH_CLIENTS = "FETCH_CLIENTS",
 FETCH_COMPANIES = "FETCH_COMPANIES",
 FETCH_INVOICES = "FETCH_INVOICES";

 export {
   FETCH_INITIAL_INFORMATION,
   FETCH_SUGGESTIONS,
   FETCH_CLIENTS,
   FETCH_COMPANIES,
   FETCH_INVOICES,
 }

```

This version has **some problems**:
1. It allows me to make mistakes. If let's say I want to remove FETCH_CLIENTS I have to remove it twice in the file (missing one of them will throw a strange error)
2. I will end up with a long file, where half of information is not useful 

The better version would be:

``` js
export const FETCH_INITIAL_INFORMATION = "FETCH_INITIAL_INFORMATION";
export const FETCH_SUGGESTIONS = "FETCH_SUGGESTIONS";
export const FETCH_CLIENTS = "FETCH_CLIENTS";
export const FETCH_COMPANIES = "FETCH_COMPANIES";
export const FETCH_INVOICES = "FETCH_INVOICES";
```

This version is _sorter_, _less prone to errors_ and _conceive_. However, my `one-var` rule collides with it, but in this way it is better to repeat the keyword.

Even the creator of Redux, [in these tutorial](https://egghead.io/courses/building-react-applications-with-idiomatic-redux) has opted for this option. Same concept goes when creating the Redux [**actions**](http://redux.js.org/docs/basics/Actions.html).

**Comment by vitorbal** [comment]
@cristian-sima thanks for the report! Could you do us a favor and fill in the [template for proposing a rule change](http://eslint.org/docs/developer-guide/contributing/rule-changes#proposing-a-rule-change)?
That will help the team understand and give feedback on your proposal faster. Thanks!

**Comment by cristian-sima** [comment]
The version of ESLint you are using
`v3.1.1`

The rule you want to change
`one-var`

The code you want to be flagged as incorrect

```js
const FETCH_INITIAL_INFORMATION = "FETCH_INITIAL_INFORMATION",
 FETCH_SUGGESTIONS = "FETCH_SUGGESTIONS",
 FETCH_CLIENTS = "FETCH_CLIENTS",
 FETCH_COMPANIES = "FETCH_COMPANIES",
 FETCH_INVOICES = "FETCH_INVOICES";

 export {
   FETCH_INITIAL_INFORMATION,
   FETCH_SUGGESTIONS,
   FETCH_CLIENTS,
   FETCH_COMPANIES,
   FETCH_INVOICES,
 }
```

What happens when the rule is applied to the code without your change

It works. But I should be allowed to both have this code 


```js
export const FETCH_INITIAL_INFORMATION = "FETCH_INITIAL_INFORMATION";
export const FETCH_SUGGESTIONS = "FETCH_SUGGESTIONS";
export const FETCH_CLIENTS = "FETCH_CLIENTS";
export const FETCH_COMPANIES = "FETCH_COMPANIES";
export const FETCH_INVOICES = "FETCH_INVOICES";
```


I better explained above

**Comment by cristian-sima** [comment]
@vitorbal done

**Comment by kaicataldo** [comment]
I'm closing this issue because it looks like consensus couldn't be reached. While we wish we could accommodate all requests, we have limited resources and do need to prioritize. We've found that issues failing to reach consensus after 21 days tend to never reach consensus, and as a team have decided to close such issues. This doesn't mean the idea isn't interesting or valuable, only that it's not something the team can commit to at the moment.
